test/puppetlabs/trapperkeeper/services/metrics/metrics_service_test.clj

```diff
@@ -67,12 +96,18 @@
                             (metrics/host-metric-name "localhost" "foo")
                             (metrics/gauge 2))
           (let [resp (http-client/get
-                      (java.net.URLDecoder/decode
                        (str "http://localhost:8180/metrics/v1/mbeans/"
-                            "pl.foo.reg:name=puppetlabs.localhost.foo")))
+                            (codec/url-encode "pl.foo.reg:name=puppetlabs.localhost.foo")))
```

Comment:
Okay cool, so long as I'm not missing something and it was previously just bizarre. As far as I can tell it doesn't actually need the url encode (or decode, for that matter), but I guess it's reasonable for it to be here to match the jolokia encode?